SPECS/inotify-tools/inotify-tools.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        Command line utility for i-node notifications and management.
 Name:           inotify-tools
-Version:        3.22.1.0
+Version:        4.23.9.0
 Release:        1%{?dist}
 License:        GPLv2
```

Comment:
Can you double check the license on this?  Probably fine, but just see if anything changed

Reply:
License did not change https://github.com/inotify-tools/inotify-tools/blob/63af779670655aa395ca1e2c70e665dc45c202a5/COPYING#L1C3-L2C19